e2e/cypress/integration/system_console/site_configuration/customization_spec.js

```diff
@@ -45,6 +45,94 @@ describe('Customization', () => {
         cy.get('#site_name').should('have.text', siteName);
         cy.get('#site_description').should('have.text', siteDescription);
     });
+
+    it('MM-T1029 - Custom branding text can render markdown text', () => {
+        // # Visit emoji system console page
+        cy.visit('/admin_console/site_config/emoji');
+        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Emoji');
+
+        // # Set Enable Custom Emoji to true
+        cy.findByTestId('ServiceSettings.EnableCustomEmojitrue').check();
+
+        // # Save setting
+        saveSetting();
+
+        // * Visit the add custom emoji page for ad-1
+        cy.visit('/ad-1/emoji/add');
+        cy.get('.backstage-header').should('be.visible');
+
+        // * Ensure the form is visible
+        cy.get('.backstage-form').should('be.visible').within(() => {
+            // # Fill the emoji data
+            cy.get('input#name').clear().type('yay');
+            cy.get('input#select-emoji').attachFile('mattermost-icon.png');
+
+            // # Save emoji
+            cy.get('.backstage-form__footer button').click();
+        });
+
+        // # Make sure necessary field is true
+        cy.apiUpdateConfig({TeamSettings: {EnableCustomBrand: true}});
+        cy.reload();
+
+        // # Visit customization system console page
+        cy.visit('/admin_console/site_config/customization');
+        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Customization');
+
+        // eslint-disable-next-line no-irregular-whitespace
+        const customBrandText = `​​​​:yay:
+
+- This is a bullet point
+- This is another bullet point
+
+**This has been bolded**
+~~This has been strikethrough~~
+*This has been italicized*
+[This is a link to mattermost.com](https://mattermost.com)
+` + '`This is inline code`'; // eslint-disable-line no-useless-concat
+
+        // # Update custom brand text
+        cy.findByTestId('TeamSettings.CustomBrandTextinput').clear().type(customBrandText);
+
+        // # Save setting
+        saveSetting();
+
+        // # Logout
+        cy.apiLogout();
+
+        // * Ensure that the user was redirected to the login page after the logout
+        cy.url().should('include', '/login');
+
+        cy.get('.signup__markdown div div').should('be.visible').within(() => {
+            // * Ensure emoji has been rendered
+            // ToDo: uncomment after fixing MM-28443
+            //cy.get('span.emoticon').should('have.attr', 'title', ':yay:');
```

Comment:
Good catch! In the ticket, could you add link to this PR so that anyone who'll on it will be able to enable this step and hopefully help validate if the fix works.  Thanks.

Reply:
Done!